Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -180,8 +182,16 @@ - (void)setupExecutionWithContext:(BPExecutionContext *)context {
     }
 
     context.runner = [self createSimulatorRunnerWithContext:context];
-
-    NEXT([self createSimulatorWithContext:context]);
+    
+    if (context.config.delDeviceID) {
```

Comment:
I guess you never used the DOS command del :-) but I am ok to rename it to the longer name "deleteDeviceID"